tests/regression/mmpose.yml

```diff
@@ -51,7 +51,7 @@ openvino:
     deploy_config: configs/mmpose/pose-detection_openvino_static-256x192.py
   pipeline_openvino_static_fp32_256x256: &pipeline_openvino_static_fp32_256x256
     convert_image: *convert_image
-    backend_test: *default_backend_test
+    backend_test: False
```

Comment:
why disable the test?

Reply:
The performance test for openvino is too long(104125 images). The conversion result is Ok on this reg test.